include/net/cloud.h

```diff
@@ -314,19 +292,15 @@ static inline int cloud_user_data_set(struct cloud_backend *const backend,
  **/
 struct cloud_backend *cloud_get_binding(const char *name);
 
-#define CLOUD_BACKEND_DEFINE(_name, _api)			               \
-									       \
-	static struct cloud_backend_config UTIL_CAT(_name, _config) =	       \
-	{								       \
-		.name = STRINGIFY(_name)				       \
-	};								       \
-									       \
-	static const struct cloud_backend _name				       \
-	__attribute__ ((section(".cloud_backends"))) __attribute__((used)) =   \
-	{								       \
-		.api = &_api,						       \
-		.config = &UTIL_CAT(_name, _config)			       \
-	};
+#define CLOUD_BACKEND_DEFINE(_name, _api)                                      \
+                                                                               \
```

Comment:
I assume the unaligned `\` at the end of lines is just GitHub messing up the indentation as usual?

Reply:
this was clang-format.  i will revert.